pkgs/tools/networking/simple-dlna-browser/default.nix

```diff
@@ -0,0 +1,57 @@
+{ pkgs, lib, ...}:
+let
+  buildInputs = with pkgs; [
+    gnused
+    socat
+  ];
+  binPaths = builtins.map (x: "${x}/bin") buildInputs;
+  paths = builtins.concatStringsSep ":" binPaths;
+in
+pkgs.stdenv.mkDerivation {
+  name = "simple-dlna-browser";
```

Comment:
Please use pname+version

Reply:
There is no version. Should I take the commit ID as the version?

Reply:
Please take the date of the commit like https://github.com/NixOS/nixpkgs/blob/ee96940bad52677b362baad3150a0931abb74350/pkgs/applications/audio/adlplug/default.nix#L49

Reply:
Done in 
0f356ab